warehouse/templates/packaging/detail.html

```diff
@@ -392,6 +392,13 @@ <h3>
 
         {% endif %}
       </div>
+    {% if release.github_repo_info_url %}
+      <div class="hidden"
+        data-controller="github-repo-stats"
+        data-github-repo-stats-github-repo-info-outlet=".github-repo-info"
+        data-github-repo-stats-url-value="{{release.github_repo_info_url}}">
+      </div>
+    {% endif %}
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏This change breaks the visual layout on desktop - resulting on placing the content in the wrong place.

<img width="869" alt="Screenshot 2023-01-08 at 3 10 11 PM" src="https://user-images.githubusercontent.com/529516/211216788-657c74bb-0424-4d6f-93ee-87261e6564bb.png">


I _think_ this wrapping `div` would have to come much earlier to apply to both controllers and not impact the layout, which adds to the complexity of this solution. 🤔

Reply:
Hm, ok.  I'll try relocating this to near the beginning of the `content` template block, I think.

Reply:
Relocated, but: I haven't yet confirmed that this displays correctly.  I'll verify that soon.

Reply:
This should now be resolved (`div` relocated in 036dd5c55038e8aecba0cd850ecb5b4c15127bf7).

(I considered using a different type of HTML element (instead of `div`) to associate with the controller - but I didn't find one that seemed to be a precise match for the use-case, and continuing to use a `div` seems straightfoward)

Reply:
Looks like the change worked!